src/3p-frame.js

```diff
@@ -65,6 +65,10 @@ function getFrameAttributes(parentWindow, element, opt_type, opt_context) {
     domFingerprint: domFingerprint(element),
     experimentToggles: experimentToggles(parentWindow),
   };
+  const layout = element.getAttribute('layout');
```

Comment:
Ditto. You need to access layout from the element via API. I see `layout_` var, but no public method - please add it.